sdk/tests/conformance2/vertex_arrays/vertex-array-object.html

```diff
@@ -476,17 +476,22 @@
     // delete the color buffers AND the position buffer.
     gl.bindVertexArray(null);
     for (var ii = 0; ii < colorBuffers.length; ++ii) {
-      gl.deleteBuffer(colorBuffers[ii]);
-      gl.deleteBuffer(elementBuffers[ii]);
       gl.bindVertexArray(vaos[ii]);
-      var boundBuffer = gl.getVertexAttrib(1, gl.VERTEX_ATTRIB_ARRAY_BUFFER_BINDING);
+
+      gl.deleteBuffer(colorBuffers[ii]);
+      gl.deleteBuffer(positionBuffer);
```

Comment:
Maybe I'm misunderstanding the spec here, but in D.1.3, it says the buffer won't be deleted while it is in use, and it defines "in use" as a buffer object being attached to a VAO

---

When a buffer, texture, sampler, renderbuffer, query, or sync object is deleted, its
name immediately becomes invalid (e.g. is marked unused), but the underlying
object will not be deleted until it is no longer in use. A buffer, texture, sampler, or
renderbuffer object is in use while it is attached to any container object (such as a
buffer object attached to a vertex array object, or a renderbuffer or texture attached
to a framebuffer object) or bound to a context bind point in any context.